src/editor/window.rs

```diff
@@ -297,7 +307,9 @@ impl Window {
 
     pub fn redraw(&self) {
         self.send_command(WindowDrawCommand::Clear);
-        for row in 0..self.grid.height {
+        // Draw the lines from the bottom up so that underlines don't get overwritten by the line
+        // below.
+        for row in self.grid.height..0 {
```

Comment:
Those two aren't quite the same. Note that `..` ranges are start inclusive, end exclusive. Also, ranges are a bit weirdly designed in Rust imo so this likely doesn't do what you'd expect, [I put a short playground example together for that](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=b771ce3e17b0215124fdbee93a978fc6). TLDR is, `start..end` is an empty iterator if `start >= end`.

Reply:
Huh, so it is. I wonder why everything still renders correctly then?